ansible/roles_ocp_workloads/ocp4_workload_rhods_handson/tasks/workload.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Clone the RHODS installation repository on bastion.
+#- name: Clone RHODS repo
+#  ansible.builtin.git:
+#    repo: git@github.com:ritzshah/olminstall.git
+#    dest: /home/lab-user/olminstall
+#    key_file: /home/lab-user/.ssh/key.pem
+
+# Clone the RHODS installation repository on bastion
+- name: Clone RHODS repo
+  git:
+    repo: 'https://{{ ocp4_workload_rhods_handson_git_access_token }}@github.com/ritzshah/olminstall.git'
+    dest: /home/cloud-user/olminstall
```

Comment:
can you please pinpoint the version?
You can use a variable for it so it can be set at the agnosticv level